openquakeplatform_ipt/static/ipt/js/consequence.js

```diff
@@ -200,6 +200,17 @@ co_obj.o.find('#downloadBtn').click(function() {
     sendbackNRML(co_obj.nrml, 'co');
 });
 
+if (typeof gem_api != 'undefined') {
+    co_obj.o.find('#delegateDownloadBtn').click(function() {
+        var uu = delegate_downloadNRML(co_obj.nrml, 'co', delegate_downloadNRML_cb);
+        console.log("fired cmd with uuid: " + uu);
```

Comment:
This console log is required ?

Reply:
We are still in development, when it will be stable enough we will remove a lot of logging.